democracy_club/templates/about/team.html

```diff
@@ -82,9 +110,22 @@ <h2>Ella Botting — Admin assistant</h2>
         </div>
     </div>
 
+   <h2>ResultsBot — Results assistant</h2>
+    <div class="ds-with-sidebar">
+        <div> <!-- inner wrapper element -->
+            <div class="ds-sidebar">
+                <img src="{% static "images/team/resultsbot.png"%}" style="width:170px;height:170px" alt="Photo of Ella Botting">
```

Comment:
Wrong sort of Botting!

Reply:
Will fix with the next design system update.